.eslintrc

```diff
@@ -1,45 +0,0 @@
-{
```

Comment:
i would very much prefer to retain the `.eslintrc` format rather than using JS for config

Reply:
Moved to `.eslintrc.json`.